### ISSUE TITLE
Turn off notifications for gists created on or after May 8th

### ISSUE THREAD

**Issue by jakebathman** [tightenco/giscus#69]
See #67 for additional info.

Any gists created on or after May 8th, 2019, should not be acted upon by Giscus.

This could probably take the form of a condition inside `IdentifiesIfACommentNeedsNotification`, since the application doesn't currently store information about the gist itself (such as its creation date).

I've not looked closely at the data we pull for each comment, but it's possible we'll need to add additional API calls to retrieve the gist creation date, since we want to continue providing new notifications for old gists.

**Comment by shayfrendt** [comment]
:wave: Ohai gist notifications friends!

I work on the notifications team at GitHub, and have to admit I first came to learn of Giscus from [this tweet](https://twitter.com/LiamHammett/status/1126197512686186496) today. There are so many amazing projects out there! I feel like I need a *Last Week Tonight with John Oliver* recap show to keep up.

I came here to say a few quick things:

1. Thank you for providing value to the world's developers by augmenting gist functionality [since 2015](https://github.com/tightenco/giscus/commit/32101dd33718858a44bad547605458e72161dc48). It's only because of passionate developers such as yourselves that GitHub continues to exist. ❤️ 

2. Now that I'm aware of your product, I'd love to loop you in closer (only if you want!) on some additional changes we're planning to help you ease the transition with your own customers:

    >since we want to continue providing new notifications for old gists.

    Specifically, we're currently reviewing how to handle notification subscriptions on old gists.

3. I'd love to meet you! Whether at [Universe](https://githubuniverse.com/) later this year, on a Zoom sometime soon, or somewhere in between, I'd love to hear more about what else you'd love to see out of GitHub's notifications ecosystem. If that sounds compelling, my email is on my profile @shayfrendt. 🙇 

I'm super duper impressed with how responsive this project is and has already landed https://github.com/tightenco/giscus/pull/68! Sorry for the fire drill if it felt like one. 

Look forward to chatting,
Shay
